packages/paste-website/src/pages/components/data-grid/index.mdx

```diff
@@ -241,6 +241,24 @@ This package provides the following component exports, which correspond closely
 You are free to put any Paste component in the Data Grid cells and it should just work.
 Keep in mind that anything that dynamically updates an element's `tabIndex` value may fail to work correctly.
 
+### Displaying missing data
+
+If the Data Grid can’t show data yet (for example, it’s in a draft state), leave cells blank.
+
+If the data underlying the Data Grid is active but the value for the cell is zero, show 0 in the cell.
+
+If a piece of data is missing because the metric in a certain column does not apply to a given row, show “N/A” and use the HTML abbreviation element to communicate the full meaning of the abbreviation (“Not applicable”). This ensures any customers using assistive technologies will understand the cell entry.
```

Comment:
Is showing the abbreviation ok if the page needs to be translated?

Reply:
Good question, no idea. Is "N/A" mainly an english speaking thing to do?

In terms of ability to translate, its fine if you do it right.

```
<abbr title="Not applicable">
  N/A
</abbr>
```

but whether or not that is appropriate or meanful localization is another matter

Reply:
If the page is translated, the translation service should know what the equivalent should be. A light google search revealed there seem to be common ways to indicate the same sentiment in german and french, so i'd imagine other languages would have corollaries. However, if we find out this isn't the case, we can update the guidance.

Reply:
sounds good!